src/main/scala/edu/ie3/simona/agent/participant/ParticipantAgentFundamentals.scala

```diff
@@ -2033,8 +2033,8 @@ object ParticipantAgentFundamentals {
           case Some(qFunc) =>
             // NOTE: The type conversion to Megawatts is done to satisfy the methods type constraints
             // and is undone after unpacking the results
-            tick -> Megawatts(qFunc(pd.toApparentPower.p).toMegavars)
-          case None => tick -> Megawatts(pd.toApparentPower.q.toMegavars)
+            tick -> Megawatts(qFunc(pd.toComplexPower.p).toMegavars)
+          case None => tick -> Megawatts(pd.toComplexPower.q.toMegavars)
```

Comment:
Do we still need the type conversion to Megawatts here?

Reply:
I think to ensure we have a value * 10^6.

Reply:
There's a type problem that is difficult to solve without being able to change the squants package, if I remember correctly.